### ISSUE TITLE
Messaging Application

### ISSUE THREAD

**Issue by cmr624** [FortyNina/You-Up#3]
Link to the profiles (for matches) and link to contact info for friends (email, phone, etc.) (?)

**Comment by elliot-winch** [comment]
My b

**Comment by cmr624** [comment]
i don't think we actually need this under messaging, this can go under tinder.
